servers.json

```diff
@@ -1175,5 +1175,13 @@
             "zoltix.net",
             "minecraftoce.net"
         ]
+    },
+    {
+        "id": "vortexnetwork",
+        "name": "Vortex Network",
+        "addresses": [
+            "vortexnetwork.net",
+            "mc.vortexnetwork.net"
```

Comment:
You do not need to add sub-domains to this array, Lunar Client services already detect them.